RELEASE-NOTES.md

```diff
@@ -39,7 +39,12 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
   `Polyhedron` object from a given primitive.
 - Adds `DataStore::getBufferInfo()` and `Group::getDataInfo` methods that insert information into a Conduit `Node` about buffers in a `DataStore` object or data in a `Group` subtree. The information can be accessed from the `Node` by the caller from specifically named fields in the `Node`.
 - Quest: Adds a `quest::ProEReader` for reading in Pro/E tetrahedral meshes
-
+- Quest: NURBS curve linearization now supports a method that can dynamically refine the curve
+  according to an error percentage. This is in addition to the existing segment-based
+  refinement method. The dynamic method is activated using `Shaper::setRefinementType()`
+  and by calling `Shaper::setPercentError()` to set a refinement error percentage. The
+  IntersectionShaper can use the percent error to determine whether the revolved volume
```

Comment:
Would it be better to move this sentence to the beginning of the change description. The error concept is mentioned multiple times before this and it may help users understand the change better if it was clear from the start what the error refers to.

Reply:
I rephrased it - hopefully for the better.